example/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@ cmake_minimum_required(VERSION 2.8)
 project(injection)
 include_directories(../ panku)
 
-set(CMAKE_CXX_FLAGS "-Wall -Werror -Wextra -std=c++14")
+set(CMAKE_CXX_FLAGS "-Wall -Werror -Wextra -std=c++14 -g")
```

Comment:
What's this for? Debug flag?

Reply:
Yeah :) only for the sample code anyway, so it shouldn't be a big deal.